doc/man7/EVP_MAC-POLY1305.pod

```diff
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-EVP_MAC-Poly1305 - The Poly1305 EVP_MAC implementation
+EVP_MAC-POLY1305 - The Poly1305 EVP_MAC implementation
```

Comment:
https://github.com/openssl/openssl/blob/89e5aaa1d72058404d3ea06bfaeff5334aba202d/providers/default/defltprov.c#L309

I think this rename should wait until we have renamed the algo

Reply:
Let's have the docs be correct instead of the code, for a change :)  The point behind changing this is that many places linked to the all-caps name and this was easier and seemed more consistent.  I am willing to open an issue for renaming those two, if it will help.

Reply:
Okay, I restored the old name.